bindings/py/cpp_src/CMakeLists.txt

```diff
@@ -63,16 +63,9 @@ set(src_py_engine_files
 set(src_py_math_files
     bindings/math/math_module.cpp    
     bindings/math/py_ArrayAlgo.cpp
-    bindings/math/py_Domain.cpp
     bindings/math/py_Math.cpp
     bindings/math/py_Random.cpp
     bindings/math/py_reals.cpp
-    bindings/math/py_Set.cpp
-    bindings/math/py_SparseBinaryMatrix.cpp
```

Comment:
Are we sure that we don't have .py code dependencies on SparseBinaryMatrix?  It would be cool if we don't need it.

Reply:
Yes, in the way that we still provide py bindings for all the algorithms (SP, TM,..) that depended on SparseBinaryMatrix (now not used at all in our c++ codebase, replaced by Connections everywhere). SM was just an internal implementation detail that should not be exposed. (in python it's far better to use native numpy matrices, and we do have binding for them too)